docs/book/features/step-fixtures.md

```diff
@@ -0,0 +1,112 @@
+# Step Fixtures
```

Comment:
Just a comment, but this is the first time I come across this term 'fixture' in ZenML. I wonder whether we want to think about whether we want to do that? Is it a common term? (Esp if we are building for data scientists)